app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            pytest.param(14, 14, [0, 0],
+                         id="Up to 15 cat/dog years give 0 human year"),
+            pytest.param(15, 15, [1, 1],
+
+                         id="First 15 cat/dog years give 1 human year"),
+            pytest.param(23, 23, [1, 1],
+                         id="24 cat/dog years should be equal 1 human years"),
+            pytest.param(27, 27, [2, 2],
+                         id="Every 4 next cat years give 1 extra human year"),
+            pytest.param(27, 28, [2, 2],
+                         id="Every 5 next dog years give 1 extra human year"),
```

Comment:
you can leave only this one without previous (27, 27) and check both use cases in one test